docs/conteudos/guia_para_mentoras.md

```diff
@@ -0,0 +1,159 @@
+
+### Não há problema em cancelar
+Se o(a) mentorado(a) estiver com muitas atividades durante a semana e não conseguirem fazer o 1:1 não tem problema, acompanhe por outros meios e veja se precisa de alguma ajuda com as atividades da BOSS ou outras. Assim como se acontecerem imprevistos e a reunião não acontecer, se organizem para a próxima (talvez alguns assuntos tenham se acumulado). 
+
+### Documento para acompanhamento
+Uma dica é usar alguma ferramenta que possa documentar os 1:1s. Desta forma pode ter um histórico para caso não se lembre de alguma informação durante o processo. Por mais que você tenha uma ótima memória, é importante ter documentado, pois uma hora você vai esquecer, e até para evitar guardar informações que podem ser facilmente guardadas de maneira organizada.
+1. Use uma ferramenta/documento compartilhado que possa ser acessado de qualquer dispositivo, caso algo ocorra de você estar com outro computador, ou até mesmo no celular. E estando online/na nuvem você pode acessar a qualquer momento, só precisando estar conectado a internet. Ou você pode ativar o modo offline (no Google Docs por exemplo é permitido).
+2. Um documento compartilhado é uma ótima maneira de guardar o que foi dito pelo(a) mentorado(a), feedbacks, ações tomadas durante a semana. Assim fica fácil pois não depende só das nossas memórias.
+3. Você pode compartilhar com o(a) mentorado(a) esses documentos para que o(a) mesmo(a) possa ver seu crescimento e histórico. E caso seja necessário crie outro documento que o(a) mentorado(a) não tenha acesso.
+
+#### Ferramentas
+
+1. [Notion](https://www.notion.so/)
+2. [Google Docs](https://docs.google.com/?hl=pt-BR)
+
+### Dois ouvidos, uma boca
+Tem um motivo para você ter dois ouvidos e uma boca. Ouvindo você aprende mais do que falando, então use isso para guiar o 1:1 e ter um resultado mais eficaz. No começo, o(a) mentorado(a) talvez não se sinta muito a vontade de falar, então vá construindo esse ambiente. Talvez para isso você tenha que falar mais até criar uma conexão entre vocês. Use dinâmicas de quebra gelo, autoconhecimento, entre outros.
+
+Este ponto fortalece a iniciativa do(a) mentorado(a) de trazer as pautas. Sugerir ferramentas para que o(a) próprio(a) mentorado(a) possa se organizar com tarefas da semana, objetivos em relação ao programa podendo trazer aos 1:1s e relatando as dificuldades e bloqueios para que você possa ajudar.
+
+Ferramentas que contemplem a metodologia kanban são um exemplo, como: [Trello](https://trello.com/), [Projects Github](https://github.com/features/project-management/)
+
+```
+“Quando você fala, está apenas repetindo aquilo que você já sabe. Mas se você escuta, então pode aprender algo novo." 
```

Comment:
eu acho uma boa citação. Mas não sei se é uma pessoa que alguém pode se sentir incomodada, por conta dessa fala dele sobre a beleza da mulher importar tanto quanto o cérebro:

> Later in our conversation I also challenged him on another remark he made in 2015, when he said if he was followed by a female Dalai Lama she would have to be attractive.
In another surprise, he reaffirmed his belief that beauty matters as much as brains. "If a female Dalai Lama comes, she should be more attractive," he told me while laughing.

(https://www.bbc.com/news/world-asia-48844825) 

Eu deixaria de fora.

Reply:
Acho válido deixar de fora.